src/server/api/routers/company.ts

```diff
@@ -0,0 +1,80 @@
+import { z } from "zod";
+import {
+  createTRPCRouter,
+  protectedProcedure,
+  publicProcedure,
+} from "~/server/api/trpc";
+import { Industry } from "@prisma/client";
+
+export const companyRouter = createTRPCRouter({
+  getAll: publicProcedure.query(({ ctx }) => {
```

Comment:
Last change: could we rename the endpoints to `list` (in place of `getAll`), `getByName`, `create`, `update`, and `delete` to keep the API minimal?